api/types/types.go

```diff
@@ -150,6 +150,7 @@ type Info struct {
 	Debug              bool
 	NFd                int
 	OomKillDisable     bool
+	MemSwappiness      bool
```

Comment:
You are right.. I think then I will also have to use it in api/types/info.go.. Do you think it is important information to be displayed?.